src/Logary.Facade/Facade.fs

```diff
@@ -315,6 +457,113 @@ module internal Formatting =
     formatExn message.fields +
     formatFields matchedFields message.fields
 
+  let literateExceptionColorizer (context: LiterateContext) (ex: exn) =
+    let stackFrameLinePrefix = "   "
+    use exnLines = new System.IO.StringReader(ex.ToString())
+    let rec go lines =
+      match exnLines.ReadLine() with
+      | null -> lines // finished reading
+      | line -> if line.StartsWith(stackFrameLinePrefix) then
+                  go (lines @ [ line, Subtext; Environment.NewLine, Text ])
+                else
+                  go (lines @ [ line, Text; Environment.NewLine, Text ])
+    go []
+
+  let literateLogLevelTokenizer = function
+    | Debug ->    "DBG", LevelDebug
+    | Error ->    "ERR", LevelError
+    | Fatal ->    "FTL", LevelFatal
+    | Info ->     "INF", LevelInfo
+    | Verbose ->  "VRB", LevelVerbose
+    | Warn ->     "WRN", LevelWarning
+
+  let literateColorizeExceptions (context: LiterateContext) message =
+    let exnExceptionParts =
+      match message.fields.TryFind FieldExnKey with
+      | Some (:? Exception as ex) ->
+        literateExceptionColorizer context ex
+        @ [ Environment.NewLine, Text ]
+      | _ -> [] // there is no spoon
+    let errorsExceptionParts =
+      match message.fields.TryFind "errors" with
+      | Some (:? List<obj> as exnListAsObjList) ->
+        exnListAsObjList |> List.collect (function
+          | :? exn as ex ->
+            literateExceptionColorizer context ex
+            @ [ Environment.NewLine, Text ]
```

Comment:
So we're doing a concat every iteration; could we do an append and List.rev in the end?